xml/Microsoft.Build.Utilities/MuxLogger.xml

```diff
@@ -22,7 +22,7 @@
       <format type="text/markdown"><![CDATA[  
```

Comment:
Line 20 here has an example of what I was saying of missing space between tags and new words.